compiler/rustc_hir_analysis/src/check/compare_impl_item.rs

```diff
@@ -1382,7 +1382,7 @@ fn compare_number_of_generics<'tcx>(
                     kind = kind,
                 ),
             );
-            err.code(DiagnosticId::Error("E0049".into()));
+            err.code("E0049".into());
```

Comment:
Drive-by: this should use the `error_code!` macro.

Reply:
I fixed that in #119972.